english/quit-your-job/quit-your-job.md

```diff
@@ -0,0 +1,5 @@
+You suddenly decide that coding and enjoying your life are far mor important than saving money. You decide that you are going to take the leap and quit your job in December 2015. As a result you must decide to 
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.